docs/source/getting-started/assessing-utility/noise.ipynb

```diff
@@ -0,0 +1,527 @@
+    "\n",
+    "This is particularly applicable when working with vectors. \n",
+    "\n",
+    "To learn more about the methodology behind the sensitivity calculation, see [chapter 6 in _Programming DP_](https://programming-dp.com/ch6.html).\n",
+    "\n",
+    "*2. There are many compositions.* \n",
+    "\n",
+    "As illustrated by the diagram, the Gaussian distribution has lighter tails. In cases with many compositions, many queries are run with independent noise. Adding more compositions leads to more dimensions to calculate the sensitivity, which leads to the case where the L2 sensitivity is likely less than the L1 sensitivity. \n",
+    "\n",
+    "#### When To Use:\n",
+    "| Laplace Noise | Gaussian Noise |\n",
+    "|-------------------|--------------------|\n",
+    "| When you need strong privacy guarantees (ε-DP). | When working with high-dimensional data or vectors. |\n",
+    "| For applications involving single numerical outputs or low-dimensional data. | For machine learning models that return outputs with many elements or complex applications that result in multidimensional data. |\n",
+    "| When the noise added should be minimal and accuracy is crucial. | When the L2 norm (Euclidean distance) is smaller than the L1 norm (sum of absolute values). |\n",
+    "| Statistical queries such as counts, sums, and averages. | When a small relaxation in privacy guarantees, i.e. (ε, δ)-DP, is acceptable in exchange for reduced noise. |\n",
+    "\n",
+    "### Summary\n",
+    "\n",
+    "- **Laplace Noise**: Best for single-value outputs and applications requiring strong privacy guarantees.\n",
+    "- **Gaussian Noise**: Best for high-dimensional data and applications where a small relaxation in privacy is acceptable for reduced noise and better utility.\n",
+    "\n",
+    "By understanding the nature of your data and the requirements of your application, you can choose the appropriate noise mechanism to ensure both privacy and accuracy.\n"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "## Specifying Laplace or Gaussian with the Context API"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "Let's get into how you could specify which type of noise you may need. The process is relatively similar to using the generic `noise()` method. The difference is that you'll specify your noise and provide the appropriate parameters. "
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 3,
+   "metadata": {},
+   "outputs": [
+    {
+     "name": "stdout",
+     "output_type": "stream",
+     "text": [
+      "  % Total    % Received % Xferd  Average Speed   Time    Time     Time  Current\n",
```

Comment:
Again, this looks untidy. Enclose it in a box or something because this is either code or some output.